src/mrdb_frontend/src/styles/loginStyle.css

```diff
@@ -0,0 +1,105 @@
+body {
+        background-color: #F3EBF6;
+        font-family: 'Ubuntu', sans-serif;
+    }
+
+    .main {
+        background-color: #FFFFFF;
+        width: 400px;
+        height: 400px;
+        margin: 7em auto;
+        border-radius: 1.5em;
+        box-shadow: 0px 11px 35px 2px rgba(0, 0, 0, 0.14);
+    }
+
+    .sign {
+        padding-top: 40px;
+        color: #8C55AA;
+        font-family: 'Ubuntu', sans-serif;
+        font-weight: bold;
+        font-size: 23px;
+    }
+
+    .un {
+    width: 76%;
```

Comment:
These styles look to be duplicated from loginPage.vue. We should avoid duplicate code. https://en.m.wikipedia.org/wiki/Duplicate_code#Costs%20and%20benefits

Reply:
I thought of making a new CSS folder which holds all the style files, but I didn't figure out how to add a source file in a vue file yet. Will make sure of this in the next sprint.

Reply:
Read about mixins in vue.